src/cromero/PrimeFactor.ts

```diff
@@ -0,0 +1,33 @@
+export const generate = (integer: number, current = []): number[] => {
+  if (integer === 1) {
+    return current;
+  }
+
+  if (isPrime(integer)) {
+    return [...current, integer];
+  }
+
+  const dividers = getDividers(integer);
+  const currentDivider = dividers[1];
+  const residue = integer / currentDivider;
+  return generate(residue, [...current, currentDivider]);
+};
+
+const getDividers = (integer): number[] => {
+  const dividers = [];
+
+  for (let i = 1; i <= integer; i++) {
+    if (integer % i === 0) {
+      dividers.push(i);
+    }
+  }
+  return dividers;
+};
```

Comment:
Is this function not the complete solution?  I mean, you receive a number and return a list of dividers (the prime factors)